FirebaseCrashlytics.podspec

```diff
@@ -58,7 +58,7 @@ Pod::Spec.new do |s|
   s.dependency 'FirebaseCore', '~> 10.5'
   s.dependency 'FirebaseInstallations', '~> 10.0'
   s.dependency 'FirebaseSessions', '~> 10.5'
-  s.dependency 'FirebaseRemoteConfigInterop', '~> 10.20'
```

Comment:
We might need to wait until after 10.22 is released and 10.23 is staged for this to keep CI working